tools/clang/test/CodeGenSPIRV/shader.debug.file.composite.hlsl

```diff
@@ -0,0 +1,30 @@
+// RUN: %dxc -E MainPs -T ps_6_0 -fspv-target-env=vulkan1.1 -fspv-debug=vulkan-with-source -spirv
```

Comment:
I believe `-spirv` is implicit for these

Reply:
OK.